### PR TITLE
Create a script to update version in each project

### PR DESCRIPTION
**Pull request by kythant**
Create a script to update version in each project. This will allow for builds to easily update the version for testing.

### CODE REVIEW

File: UpdateVersions.ps1 [hunk 1]
```diff
@@ -0,0 +1,40 @@
+Param(
+    [string]$WinAppSDKVersion = "",
```
Comment (erik0686) [inline UpdateVersions.ps1:2]: Where are we getting this variable from?

Comment (kythant) [reply UpdateVersions.ps1:2]: From the caller :)

Comment (erik0686) [reply UpdateVersions.ps1:2]: Sorry, I missed the %* in the cmd file

Comment (kythant) [reply UpdateVersions.ps1:2]: Specifically, if the caller was our nightly build, the nightly build will have the version variable that it used for packing the nuget stored somewhere

**Review by erik0686 — APPROVED** [review]

**Pull request merged**